src/crawlee/crawlers/_playwright/_types.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from typing import Protocol
+
+from crawlee._utils.docs import docs_group
+
+
+@docs_group('Functions')
+class BlockRequestsFunction(Protocol):
+    """A function for blocking unwanted HTTP requests during page loads in PlaywrightCrawler.
+
+    It simplifies the process of blocking specific HTTP requests during page navigation.
+    The function allows blocking both default resource types (like images, fonts, stylesheets) and custom URL patterns.
+
+    Default patterns for blocking ['.css', '.webp', '.jpg', '.jpeg', '.png', '.svg', '.gif', '.woff', '.pdf', '.zip']
```

Comment:
Oh,  this is risky as it could become easily outdated if we update the value of `_DEFAULT_BLOCK_REQUEST_URL_PATTERNS`. I would prefer not to mention it here, or alternatively, we could consider inlining the variable's value directly into the docstring, if it is possible.